### ISSUE TITLE
pyKAN list_modules crashes with TypeError

### ISSUE THREAD

**Issue by minecraft2048** [ajventer/pyKAN#31]
I'm running fresh pyKAN master cloned from this repository, and this happens:
```
feanor@silmaril ~/D/P/pyKAN> ./pyKAN list_modules 
Using KSP Directory:  /home/feanor/Development/Kerbal Space Program
Traceback (most recent call last):
  File "./pyKAN", line 404, in <module>
    options.func(options)
  File "./pyKAN", line 164, in list_modules
    if modid not in result or Version(result[modid]['version']) < Version(version):
  File "/home/feanor/Development/Programming/pyKAN/libPyKAN/version.py", line 111, in __lt__
    return self.__cmp__(other) < 0
  File "/home/feanor/Development/Programming/pyKAN/libPyKAN/version.py", line 81, in __cmp__
    return -other.__cmp__(self)
  File "/home/feanor/Development/Programming/pyKAN/libPyKAN/version.py", line 104, in __cmp__
    if i > j:
TypeError: '>' not supported between instances of 'int' and 'str'
feanor@silmaril ~/D/P/pyKAN> 
```

I think that this commit ad4e4f4407cad0590d7a06941d6b66b5e6fc5cd5 is the problem, as you fixed incorrect version comparison by trying to convert it to integers, and for some reason it fails to convert to integer for one of the version numbers

**Comment by ajventer-tal** [comment]
That's strange. I'm using that same version daily at home and I haven't had this issue. Though I rarely use the list feature - so it could be a value that changed in the upstream data recently that triggered this. I'll investigate it tonight and see if I can post a fix.

**Comment by ajventer** [comment]
Fixed in commit: ba8a8e3956ad0183bc9d5a4f2b79c01fcdcaf960
